lib/iris/tests/test_nimrod.py

```diff
@@ -87,7 +94,14 @@ def test_load_kwarg(self):
             "Ellipsoid not supported, proj_biaxial_ellipsoid:-32767, horizontal_grid_type:0",
         ):
             with open(
-                tests.get_data_path(("NIMROD", "uk2km", "cutouts", datafile,)),
+                tests.get_data_path(
+                    (
+                        "NIMROD",
+                        "uk2km",
+                        "cutouts",
+                        datafile,
+                    )
+                ),
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?